### ISSUE TITLE
Allow enabling / disabling disconnected agents

### ISSUE THREAD

**Issue by pavelsher** [bunopus/Teamcity.DisableAllAgents#2]
Currently plugin works with connected agents only (BuildAgentManager.getRegisteredAgents()) but links to enable / disable are shown on disconnected tab too, so it makes sense to make plugin work for disconnected agents too (BuildAgentManager.getUnregisteredAgents).

**Comment by bunopus** [comment]
@pavelsher Thank you! But is there any sence in disabling disconnected agents? Are they will stay disabled after connection?

**Comment by pavelsher** [comment]
Yes it make sense. Agent can disconnect for short time, upgrade, OS reboot, network problems, etc. And yes it will remain disabled after connection.
